assignment_04/app/Contracts/Dao/MajorDaoInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
Check previous assignment PR review in this assignment and fix if still incorrect.
**For the whole project files**